assets/js/romo/select_dropdown.js

```diff
@@ -6,7 +6,8 @@ $.fn.romoSelectDropdown = function(optionElemsParent) {
 
 var RomoSelectDropdown = function(element, optionElemsParent) {
   this.elem = $(element);
-  this.defaultCaretClass = '';
+  this.defaultCaretClass   = undefined;
+  this.defaultCaretWidthPx = 0;
```

Comment:
@kellyredding - Shouldn't these be on the select component and not the select dropdown? In the select js diff, we use `this.defaultCaretClass` and `this.defaultCaretWidthPx` and `this` seems to reference an instance of a select component not a select dropdown component.